.github/workflows/macos_tests.yml

```diff
@@ -153,7 +165,15 @@ jobs:
         with:
           persist-credentials: false
           submodules: true
+      - name: Swift build
+        run: |
+          if [ -n "${{ matrix.config.build_arguments_override }}" ]; then
+            swift build "${{ matrix.config.build_arguments_override }}"
```

Comment:
Just checking that `--build-tests` is purposefully dropped here because the args are being overridden?

Reply:
Yeah I thought giving people power to totally customize was more general.